mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -481,12 +481,18 @@ pub fn btc_segwit_conf() -> Json {
         "txfee": 0,
         "estimate_fee_mode": "ECONOMICAL",
         "mm2": 1,
-        "enable_spv_proof": true,
-        "block_headers_verification_params": {
-            "difficulty_check": true,
-            "constant_difficulty": false,
-            "difficulty_algorithm": "Bitcoin Mainnet",
-            "genesis_block_header": "010000006fe28c0ab6f1b372c1a6a246ae63f74f931e8365e15a089c68d6190000000000982051fd1e4ba744bbbe680e1fee14677ba1a3c3540bf7b1cdb606e857233e0e61bc6649ffff001d01e36299"
+        "spv_conf": {
+            "block_header": {
+                "height": 0,
+                "hash": "000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f",
+                "bits": 486604799,
+                "time": 1231006500,
+            },
+            "validation_params": {
+                "difficulty_check": true,
+                "constant_difficulty": false,
+                "difficulty_algorithm": "Bitcoin Mainnet"
+            }
```

Comment:
No need to add `spv_conf` to `btc_segwit_conf`. This will increase the test times if somebody chose to use this config in a test which activate `btc-segwit` using v2 activation methods.